compiler_test.go

```diff
@@ -229,8 +229,8 @@ message Foo {
 	})
 
 	compiler := Compiler{
-		Resolver:          WithStandardImports(&SourceResolver{Accessor: accessor}),
-		IncludeSourceInfo: true,
+		Resolver:       WithStandardImports(&SourceResolver{Accessor: accessor}),
+		SourceInfoMode: SourceInfoStandard,
```

Comment:
It might be good to add an end to end test that uses a difference SourceInfoMode that verifies the issue upstream in Buf CLI.

Reply:
The new test in `sourceinfo` does that.